client/trino-jdbc/src/main/java/io/trino/jdbc/TrinoConnection.java

```diff
@@ -131,9 +137,17 @@
     @Override
     public Statement createStatement()
             throws SQLException
+    {
+        return createTrinoStatement();
+    }
+
+    private TrinoStatement createTrinoStatement()
```

Comment:
current name could be misunderstood as meaning "create some other kind of statement than `createStatement()` would do"
`doCreateStatement` would convey the actual meaning better